portfolio/src/main/webapp/index.html

```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
             <center>
               <h2>Courses</h2>
               <p class="struct">
-                <!-- TODO: Change to unordered list & keep consistent formatting -->
-                Data Structures and Algorithms (CS 225)<br>
-                Computer Architecture (CS 233)<br>
-                Discrete Structures (CS 173)<br>
-                Numerical Methods (CS 357)<br>
-                Intro to Computer Science (CS 125) <br>
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/courses.png" height="250" width="250"/></center>
+                Data Structures and Algorithms (CS 225)<br />
+                Computer Architecture (CS 233)<br />
+                Discrete Structures (CS 173)<br />
+                Numerical Methods (CS 357)<br />
+                Intro to Computer Science (CS 125) <br />
+                </div>
+              </p>
+            </center>
+            <div class="image">
+              <center><img class="img-responsive" src="images/courses.png" height="250" width="250" /></center>
+            </div>
           </div>
         </div>
       </div>
-    </div>
-    <!-- Hobbies -->
-    <div class="info-row info-row1">
-      <div class="container">
-        <div class="row">
-          <div class="about">
-            <center>
-              <h2>Hobbies</h2>
-              <p class="struct">
-                <!-- TODO: Change to unordered list & keep consistent formatting -->
-                Making DIY Projects<br>
-                Reading Self-Improvement Books<br>
-                Oil Painting<br>
-                Sipping Herbal Tea<br>
-              </p>
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/hobbies.png" height="250" width="250"/></center>
+      <!-- Hobbies -->
+      <div class="info-row info-row1">
+        <div class="container">
+          <div class="row">
+            <div class="about">
+              <center>
```

Comment:
Supposedly, center isn't permitted inside a div. If you remove it, does it actually change anything?